src/components/context-menu-buttons/ChainContextMenu.tsx

```diff
@@ -61,7 +61,7 @@ export const ChainContextMenu = ({
         actionTitle: chainsLabel[chainId],
         icon: {
           iconType: 'ASSET',
-          iconValue: `${chainsName[chainId]}Badge${chainId === ChainId.mainnet ? '' : 'NoShadow'}`,
+          iconValue: chainId === ChainId.mainnet ? 'ethereumBadge' : `${chainsName[chainId]}BadgeNoShadow`,
```

Comment:
do we have a `ethereumBadgeNoShadow` icon? if not we could change the icon name to keep a standard of being the chain name

Reply:
we don't but I am planning on reworking these soon to be backend driven anyway so not going to bother on doing this now.